src/garage/sampler/utils.py

```diff
@@ -111,6 +115,6 @@ def truncate_paths(paths, max_samples):
                 truncated_last_path[k] = tensor_utils.truncate_tensor_dict(
                     v, truncated_len)
             else:
-                raise NotImplementedError
+                raise NotImplementedError()
```

Comment:
can you raise a ValueError instead? and include an error message which specifies the valid keys versus the ones found?